cvpy/annotation/base/AnnotationLabel.py

```diff
@@ -19,3 +19,13 @@ def color(self) -> str:
     @color.setter
     def color(self, color) -> None:
         self._color = color
+        
+    def as_dict(self) -> dict:
+        '''
+        Creates a dictionary representation of this object.
+        :return: A dictionary with all of the properties as keys and the property values as values
+        '''
+        d = {}
+        for k, v in vars(self).items():
+            d[k[1:]] = v
+        return d
```

Comment:
This file and some other files don't have a newline at the end of the file. The python coding standard PEP8 recommends adding a new line at the end of a file (see W292 on https://pep8.readthedocs.io/en/release-1.7.x/intro.html). 

In general, it is a good practice to add a newline at the end of a text file. 
https://stackoverflow.com/questions/5813311/whats-the-significance-of-the-no-newline-at-end-of-file-log

Reply:
Done. I've also run the linter and fixed as many other minor issues as possible.